src/components/common/input/NewAppInput.tsx

```diff
@@ -26,6 +69,15 @@ interface NewAppInputProps extends React.InputHTMLAttributes<HTMLInputElement> {
   enterHandler?: () => void;
   width?: number;
   height?: number;
+  label?: string;
+  buttonProps?: {
+    size?: CustomButtonSize;
+    buttonText: string;
+  };
+  linkProps?: {
+    linkText: string;
+    linkUrl: string;
```

Comment:
```suggestion
    text: string;
    url: string;
```
link는 굳이 안써도 될 것 같아요